tests/helpers/test_script.py

```diff
@@ -5807,6 +5812,37 @@ def broken_service(service: ServiceCall) -> None:
     )
 
 
+async def test_actions_enabled_template_limited(hass: HomeAssistant) -> None:
```

Comment:
The test function name is a bit misleading, this is a negative test to ensure only limited templates are allowed and the error reporting is sufficient right?

Reply:
Renamed to `test_enabled_error_non_limited_template`. If you have another suggestion, let me know... naming is hard 🙃